jb-yoda/Dockerfile

```diff
@@ -28,4 +28,4 @@ RUN ["mvn", "package"]
 EXPOSE $YODA_RPC_PORT
 
 # Make default entrypoint start Yoda's gRPC server
-ENTRYPOINT ["mvn", "exec:java", "-Dexec.mainClass=life.jukebox.yoda.YodaGrpcServer"]
+ENTRYPOINT ["mvn", "exec:java", "-Dexec.mainClass=life.jukebox.yoda.YodaSubscriberClient"]
```

Comment:
Please change the entrypoint backtoYodaGrpcServer. When the service starts, we still should start the gRPC server. In the future, we will most likely have a Yoda.class that starts the clients for pubsub, datastore, etc. and then start the gRPC server. But for now, let's just start the gRPC servr.

Reply:
?